common/supabase/schema.ts

```diff
@@ -69,6 +69,23 @@ export interface Database {
           fs_updated_time?: string
         }
       }
+      contract_embeddings: {
```

Comment:
This is for the OpenAI semantic embeddings and has nothing to do with this PR, or am I very mistaken?

Reply:
Yes it's generated from the database so doesn't really matter all that much